conf/scripts/init.sc

```diff
@@ -24,7 +27,7 @@ import org.apache.spark.rdd._
     val envJars = sys.env.get("ADD_JARS")
     val propJars = sys.props.get("spark.jars").flatMap { p => if (p == "") None else Some(p) }
     val jars = List(propJars, envJars).collect{case Some(j) => j}.mkString(",")
-    notebook.Utils.resolveURIs(jars).split(",").filter(_.nonEmpty)
+    resolveURIs(jars).split(",").filter(_.nonEmpty)
```

Comment:
I would worry about initialization order here.

`resolveURIs` uses `isWindows` & co. `val`s which are defined later than being used, and this happens inside of an initializer of `new java.io.Serializable {` class. I'd except for trouble :trollface: